test/verifiedTests/testTopology/testFindExtremePathway.m

```diff
@@ -46,14 +42,14 @@
     solverOK = changeCobraSolver(solverPkgs{k}, 'LP', 0);
 
     if solverOK == 1
-        v = findExtremePathway(model);
-        assert(all(abs(model.S * v) < tol))
-
-        % testing findExtremePathway with different arguments
-        if isfield(model, 'revRxns')
-            model = rmfield(model, 'revRxns');
-        end
-
+%         v = findExtremePathway(model);
```

Comment:
it is essentially obsolete, as it is the same as the later tests, since revRxns and reversibilities were removed as model fields.